docs/hive.md

```diff
@@ -38,6 +38,16 @@ Iceberg compatibility with Hive 2.x and Hive 3.1.2/3 supports the following feat
 DML operations work only with MapReduce execution engine.
 {{< /hint >}}
 
+With Hive version 4.0.0-alpha-2 and above,
```

Comment:
Why does this refer to an alpha release? I think we should only reference official Hive releases, and probably not alpha ones.

Reply:
Hive is pushing for the final release. There were no new releases for a few years now, but the community started to finalize one. And one of the main feature there is the native Iceberg support which is not there with older releases. We need to notify the commit about the new features to gather feedback and fix issues for the final release.